source/gui/settingsDialogs.py

```diff
@@ -40,17 +40,20 @@
 	updateCheck = None
 import inputCore
 import nvdaControls
+import winVersion
 
 class SettingsDialog(wx.Dialog):
 	"""A settings dialog.
-	A settings dialog consists of one or more settings controls and OK and Cancel buttons.
-	Action may be taken in response to the OK or Cancel buttons.
+	A settings dialog consists of one or more settings controls and OK and Cancel buttons and an optional Apply button.
+	Action may be taken in response to the OK, Cancel or Apply buttons.
 
 	To use this dialog:
 		* Set L{title} to the title of the dialog.
+		* Set L{settingsSizerOrientation} to one of wx.VERTICAL or wx.horizontal to set the orientation of the settings sizer.
+		* Set L{hasApplyButton} to C{True} to add an apply button to the dialog.

```

Comment:
I think perhaps here to the docs for `hasApplyButton`: "defaults to C{False} for in SettingsDialogs backwards compatibility."